digital_land/harmonise.py

```diff
@@ -153,12 +152,13 @@ def harmonise(self, reader):
 
             for field in row:
                 row[field] = self.apply_patch(field, row[field])
-                if self.harmonise_callback and hasattr(
-                    self.harmonise_callback, "patch"
-                ):
-                    row[field] = self.harmonise_callback.patch(
-                        field, row[field], self.log_issue
+                if self.plugin_manager:
+                    plugin_results = self.plugin_manager.hook.apply_patch_post(
+                        fieldname=field, value=row[field]
                     )
+                    if len(plugin_results) == 1:
```

Comment:
It can return an array?

Reply:
yes, `plugin_manager.hook.<function>` will always return a list, as you might have multiple implementations for a given hook.